zerver/models/realms.py

```diff
@@ -808,6 +808,7 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
     )
 
     REALM_PERMISSION_GROUP_SETTINGS_WITH_NEW_API_FORMAT = [
+        "create_multiuse_invite_group",
```

Comment:
For this, we should also update `require_system_group` field in `GroupPermissionSetting` object.

Reply:
I don't think we would change the UI for `can_access_all_users_group` as I guess we would need to do some more work to allow restricting user access for non-guest users. So, this change should not be done for `can_access_all_users_group` setting.

Reply:
I guess the rule for what the implementation supports is that `can_access_all_users_group` must contain either the MEMBERS or EVERYONE role group in its group-setting value. 

For now we can keep the UI as-is and use `allowed_system_groups` to make it a group-setting value whose value can only be those 2 system groups. But later we'd need to change the `GroupPermissionSetting` type to support specifying what is required for that setting. Doesn't seem urgent.

Reply:
Fixed!